content/concepts/dolt/sql/views.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Views
+---
+
+# Views
+
+## What is a View?
+
+Views look like tables when used in a select statement but the data in views is generated using SQL stored in the view definition. The data is actually stored in the tables the views reference not the view itself.
+
+## How to use Views
+
+Views allow you to derive tables using SQL instead of storing a copy of all the data you might want to derive. As a simple example, in a table of employee's salaries, you may store yearly salary but when using the table to calculate monthly salary, you use a view that divides the yearly salary by 12.
+
+## Difference between MySQL Views and Dolt Views
+
+There is no difference between MySQL and Dolt views. They are functionally equivalent.
+
+## Interaction with Dolt Version Control
+
+Dolt view definitions are versioned in the `dolt_schemas` system table. 
+
+If you would like to use a current view with an different version of the data, `as of` syntax works with views.
```

Comment:
the current working set's view? this one wasn't completely obvious to me, so if the view is different in the other root we still use this root's view definition?